ml-proto/test/block.wast

```diff
@@ -200,7 +200,7 @@
 
 ;; TODO(stack): move this elsewhere
 (module (func $type-break-num-vs-void
-  (block (i32.const 66) (br 0))
+  (block (drop (i32.const 66)) (br 0))
```

Comment:
At one point at least the AST did support code such as `(i32.add (i32.const 1) (br ...))` and `(block (i32.const 1) (i32.const 2))` and 'statements' are consistent with an AST.